src/main/java/de/cowtipper/cowlection/listener/skyblock/DungeonsListener.java

```diff
@@ -360,7 +362,13 @@ private void renderPartyStatus(ItemStack item, int x, int y) {
                 } else if (" Empty".equals(toolTipLineWithoutFormatting)) {
                     --partySize;
                 } else if (toolTipLineWithoutFormatting.startsWith("Note: ")) {
-                    String partyNote = toolTipLineWithoutFormatting.toLowerCase();
+                    String partyNote = toolTipLineWithoutFormatting.toLowerCase().trim();
+                    while (i + 1 < itemTooltip.size()) {
+                        String next = EnumChatFormatting.getTextWithoutFormattingCodes(itemTooltip.get(i + 1)).trim().toLowerCase();
+                        if (next.isEmpty()) break;
+                        partyNote += " " + next;
+                        i++;
+                    }
```

Comment:
Something like this should work:

```suggestion
                } else if (toolTipLineWithoutFormatting.startsWith("Note:")) {
                    String partyNote = toolTipLineWithoutFormatting.substring(5 /* remove "Note:" */).trim().toLowerCase();

                    if (i + 1 < itemTooltip.size()) {
                        String nextToolTipLine = itemTooltip.get(i + 1);
                        if (!nextToolTipLine.isEmpty() && (nextToolTipLine.startsWith("§7§f") || nextToolTipLine.startsWith("§f"))) {
                            partyNote += " " + EnumChatFormatting.getTextWithoutFormattingCodes(nextToolTipLine).trim().toLowerCase();
                            i++;
                        }
                    }
```